brewtils/decorators.py

```diff
@@ -108,16 +108,31 @@ def echo_json(self, message):
             from decorators and inferred from the method signature.
         command_type: The command type. Valid options are Command.COMMAND_TYPES.
         output_type: The output type. Valid options are Command.OUTPUT_TYPES.
-        schema: A custom schema definition.
-        form: A custom form definition.
-        template: A custom template definition.
+        schema: Deprecated and will be removed in future release. Custom schema definition.
+        form: Deprecated and will be removed in future release. Custom form definition.
+        template: Deprecated and will be removed in future release. Custom template definition.
         icon_name: The icon name. Should be either a FontAwesome or a Glyphicon name.
         hidden: Flag controlling whether the command is visible on the user interface.
         metadata: Free-form dictionary
 
     Returns:
         The decorated function
     """
+    if form is not None:
```

Comment:
I noticed when started beer-garden that these deprecation warnings actually show up twice, I think because I had multiple decorators.  I think if you move these checks into the `if _wrapped is None:` block the warning will only ever show up once.   To duplicate this just add one of these parameters to a `@command` anywhere that `@parameter` is also used in a plugin.  Then start the garden and look at the log output.  For example, from the echo example plugin:

```python
    @command(output_type="JSON", template="cool")
    @parameter(
        key="message",
        type="String",
        description="The Message to be Echoed",
        optional=True,
        default='{"str": "value", "nums": [1, 17], "obj": {"nested": "sweet"}}',
    )
    def say_json(self, message):
    """Echos with JSON output_type"""
    return message
```

Doing this showed the warning twice for me, with the second warning pointing to the `@parameter` line, which could be somewhat confusing.